modules/terrain/src/lib/parse-quantized-mesh.js

```diff
@@ -43,9 +44,23 @@ function getTileMesh(arrayBuffer, options) {
   }
   const {bounds} = options;
   // Don't parse edge indices or format extensions
-  const {header, vertexData, triangleIndices} = decode(arrayBuffer, DECODING_STEPS.triangleIndices);
+  const {
+    header,
+    vertexData,
+    triangleIndices: originalTriangleIndices
+  } = decode(arrayBuffer, DECODING_STEPS.triangleIndices);
```

Comment:
```suggestion
    westIndices,
    northIndices,
    eastIndices,
    southIndices
  } = decode(arrayBuffer, DECODING_STEPS.edgeIndices);
```
This is where you get the edge indices from the quantized mesh https://github.com/visgl/loaders.gl/blob/vb/terrain-skirting/modules/terrain/src/lib/decode-quantized-mesh.js